### PR TITLE
Adds 2019 libraries

### PR DESCRIPTION
**Pull request by ThadHouse**
Uses development for now until we have a beta release published to release maven.

### CODE REVIEW

File: src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIPlugin.groovy [hunk 1]
```diff
@@ -39,15 +39,5 @@ class WPIPlugin implements Plugin<Project> {
                 }
             }
         }
-
-        // TODO: Remove for stable, update for 2019 corelibs when appropriate
-        project.afterEvaluate {
-            def style = StyledTextOutput.Style.SuccessHeader
```
Comment (JacisNonsense) [inline src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIPlugin.groovy]: Should we keep this here, but change it to beta?

Comment (ThadHouse) [reply src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIPlugin.groovy]: We can. But since this is 2019 libs were not out of sync anymore.

Comment (JacisNonsense) [reply src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIPlugin.groovy]: I think a warning along the lines of: 
```
NOTE: You are using a BETA version of GradleRIO, designed for the 2019 Season!
This release requires the 2019 RoboRIO Image, and may be unstable. Do not use this for the official competition season. 
If you encounter any issues and/or bugs, please report them to https://github.com/wpilibsuite/GradleRIO
```

Comment (ThadHouse) [reply src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIPlugin.groovy]: Ok, Fixed

**Pull request merged**